universal-login-react/src/ui/Notifications/EmojiForm.tsx

```diff
@@ -30,10 +30,14 @@ export const EmojiForm = ({deployedWallet, hideTitle, className, onDenyRequests,
   const [gasParameters, setGasParameters] = useState<GasParameters | undefined>(undefined);
 
   const [notifications, setNotifications] = useState<Notification[]>([]);
-  useEffect(() => deployedWallet.subscribeAuthorisations(setNotifications), []);
+  const [soleAddress, setSoleAddress] = useState<string | undefined>(undefined);
```

Comment:
Maybe we should add a default value to soleAddress and check for this value. I don't like that default value is undefined and later you set this value to undefined if addresses array is bigger than 1.